.circleci/scripts/binary_linux_build.sh

```diff
@@ -16,19 +16,19 @@ if [[ "${DESIRED_CUDA}" == "cu111" || "${DESIRED_CUDA}" == "cu113" ]]; then
 fi
 
 # Parse the parameters
-if [[ "$PACKAGE_TYPE" == 'conda' ]]; then
+if [[ "${PACKAGE_TYPE}" == 'conda' ]]; then
```

Comment:
Minor nit, we should probably save these types of cleanup changes for separate PRs, just in case someone is looking at the diff in the future and is confused why this change is tied to this particular commit

Reply:
done